web-client/src/modules/settings/components/SettingsList/SettingsList.tsx

```diff
@@ -118,8 +120,8 @@ const SettingsList: React.FC<SettingsProps> = ({
             <span>{t('settings.deleteAccount')}</span>
           </SettingsListButton>
         </Col>
-      </Row>
-      <Row gutter={[0, 12]}>
+      </SettingsListPopupWrapper>
+      <SettingsListItemWrapper>
```

Comment:
You see here, I needed to add another item here for changing address which behaves similarly to how the change name component works.
Having another CSS just for address with hardcoded values for margin here wouldn't be the right approach, instead it should work automatically.